samples/server/petstore/php-slim4/composer.json

```diff
@@ -32,6 +32,9 @@
     "psr-4": { "OpenAPIServer\\": "tests/" }
   },
   "scripts": {
+    "post-install-cmd": [
+      "php -r \"chmod('./logs', 0777);\""
```

Comment:
Not a security expert would that (making log readable and writable by anyone) introduce any security concern?

Reply:
Hmm, you're right. I use this mask most of the time because default Apache configuration starts server under `www` user, not the `root` one. And if I understand correctly `www` is part of the `group` users or maybe even `global`. That's why I've encountered permission error within `logs` folder many times before. So we should change the permission mask to something more safe like `0644`(user read/write, others only read). But that's might be default mask when you create new directory or a file then command above doesn't make sense. Let's close this PR without merge...

Reply:
Please leave this open. I've some ideas...

Reply:
Just checked default Apache config, likely I've installed latest version via Homebrew just few days ago:
```apache
<IfModule unixd_module>
#
# If you wish httpd to run as a different user or group, you must run
# httpd as root initially and it will switch.  
#
# User/Group: The name (or #number) of the user/group to run httpd as.
# It is usually good practice to create a dedicated user and group for
# running httpd, as with most system services.
#
User _www
Group _www

</IfModule>
```
then when I generate files/directories I write them obviously from my superuser(`root`), but Apache works under `_www` user and it doesn't have write permission to `logs` folder. The same problem when developer wants to move user uploads from temp folder(Apache puts them into temp by default) to projects specific directory, they usually got `Permission denied` error. Then the same `sudo chmod -R a+w ./uploads` scenario as workaround.